app/src/main/java/com/kickstarter/libs/utils/RewardUtils.kt

```diff
@@ -0,0 +1,209 @@
+package com.kickstarter.libs.utils
+
+import android.content.Context
+import android.util.Pair
+import com.kickstarter.R
+import com.kickstarter.libs.KSString
+import com.kickstarter.libs.models.OptimizelyExperiment
+import com.kickstarter.models.Project
+import com.kickstarter.models.Reward
+import org.joda.time.DateTime
+import org.joda.time.Duration
+import kotlin.math.floor
+import kotlin.math.max
+
+object RewardUtils {
+
+    /**
+     * Returns `true` if the reward has backers, `false` otherwise.
+     */
+    fun hasBackers(reward: Reward): Boolean {
+        return IntegerUtils.isNonZero(reward.backersCount())
+    }
+
+    fun isAvailable(project: Project, reward: Reward): Boolean {
+        return project.isLive && !isLimitReached(reward) && !isExpired(reward)
+    }
+
+    /**
+     * Returns `true` if the reward has expired.
+     */
+    fun isExpired(reward: Reward): Boolean {
+        return isTimeLimited(reward) && reward.endsAt()!!.isBeforeNow
```

Comment:
Use un here the unwrap technique with elvis operator, do not force an option with the !! operator 
`reward.endsAt()?.let { it.isBeforeNow } ?: false`